src/library_glfw.js

```diff
@@ -137,6 +137,8 @@ var LibraryGLFW = {
         case 0xAD:return 45; // DOM_VK_HYPHEN_MINUS -> GLFW_KEY_MINUS
         case 0xBE:return 46; // DOM_VK_PERIOD -> GLFW_KEY_PERIOD
         case 0xBF:return 47; // DOM_VK_SLASH -> GLFW_KEY_SLASH
+        case 0xBD:return 45; // Opera, IE, Safari, Chrome DOM_VK_MINUS -> GLFW_KEY_MINUS
```

Comment:
Thank you.

Changes are in 2bb29de3ac520ad6fc07180a72428175939fa6fb.  What is my next step?